src/gui/tray/ActivityItem.qml

```diff
@@ -0,0 +1,288 @@
+import QtQml 2.12
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import Style 1.0
+
+MouseArea {
+    id: activityMouseArea
+    enabled: (path !== "" || link !== "")
+    width: Style.trayWindowWidth
+    height: Style.trayWindowHeaderHeight
+    Layout.alignment: Qt.AlignLeft
+    Layout.margins: 2
+    Layout.fillWidth: true
+    Layout.fillHeight: true
+    hoverEnabled: true
+    onClicked: activityModel.triggerDefaultAction(model.index)
+    
+    Rectangle {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        width: Style.trayWindowWidth
+        height: Style.trayWindowHeaderHeight
+        color: (parent.containsMouse ? Style.lightHover : "transparent")
+        
+        RowLayout {
+            id: activityItem
+            Layout.fillWidth: true
```

Comment:
you are not within a layout, you should not use `Layout` attached properties and you probably want to use `anchors.fill: parent`

Reply:
If I use anchors here I get "qrc:/qml/src/gui/tray/ActivityItem.qml:106:13: QML RowLayout: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead."

Reply:
Nevermind, wrong RowLayout.